go/chat/search/indexer.go

```diff
@@ -53,13 +59,14 @@ func NewIndexer(g *globals.Context) *Indexer {
 	idx := &Indexer{
 		Contextified: globals.NewContextified(g),
 		DebugLabeler: utils.NewDebugLabeler(g.GetLog(), "Search.Indexer", false),
-		store:        newStore(g),
 		pageSize:     defaultPageSize,
-		stopCh:       make(chan chan struct{}, 10),
+		stopCh:       make(chan struct{}),
 		suspendCh:    make(chan chan struct{}, 10),
 		resumeWait:   time.Second,
 		cancelSyncCh: make(chan struct{}, 100),
 		pokeSyncCh:   make(chan struct{}, 100),
+		clock:        clockwork.NewRealClock(),
+		flushDelay:   15 * time.Second,
```

Comment:
why not higher?

Reply:
This felt fine to me, any higher and I would like to put in a response to mobile app state,.